### ISSUE TITLE
Initial implementation

### ISSUE THREAD

**Issue by g4s8** [artipie/helm-adapter#1]
Add documentation about Helm repository,
create metadata generators,
create HTTP API,
add integration tests.

Blocked by #8

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) I'm not managing this repo, remove the [webhook](https://github.com/artipie/helm-adapter/settings/hooks) or contact me in Slack, as explained in [§11](http://www.zerocracy.com/policy.html#11); I'm not managing `artipie/helm-adapter` GitHub repository, you have to contact me in Slack first, see [§11](http://www.zerocracy.com/policy.html#11) /cc @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by 0crat** [comment]
I'm not managing `artipie/helm-adapter` GitHub repository, you have to contact me in Slack first, see [§11](http://www.zerocracy.com/policy.html#11)

**Comment by Sammers21** [comment]
@g4s8, assign the issue to me

**Comment by 0crat** [comment]
I'm not managing `artipie/helm-adapter` GitHub repository, you have to contact me in Slack first, see [§11](http://www.zerocracy.com/policy.html#11)

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@65475cc8-d3fa-45ad-ada7-34879af0f613, thread: pool-1165-thread-1
com.zerocracy.radars.github.ThrottledComments[74] java.lang.IllegalStateException: Can't post an identical message to artipie/helm-adapter#1: I'm not managing `artipie/helm-adapter` GitHub repository, you have to contact me in Slack first, se
```

[0.53.14](https://github.com/zerocracy/farm): Issue: artipie/helm-adapter#1, Action: `assigned`

**Comment by Sammers21** [comment]
Implementation progress:

- [ ] index.yml generation

**Comment by g4s8** [comment]
@rultor release, tag=`0.1`

**Comment by g4s8** [comment]
@rultor release, tag=`0.1.1`

**Comment by 0crat** [comment]
I'm not managing `artipie/helm-adapter` GitHub repository, you have to contact me in Slack first, see [§11](http://www.zerocracy.com/policy.html#11)

**Comment by g4s8** [comment]
@rultor release, tag=`0.1.2`

**Comment by g4s8** [comment]
@rultor release, tag=`0.1.2`

**Comment by g4s8** [comment]
@rultor release, tag=`0.1.2`

**Comment by 0pdd** [comment]
@g4s8 the puzzle [#44](https://github.com/artipie/helm-adapter/issues/44) is still not solved.

**Comment by 0pdd** [comment]
@g4s8 2 puzzles [#44](https://github.com/artipie/helm-adapter/issues/44), [#52](https://github.com/artipie/helm-adapter/issues/52) are still not solved.

**Comment by 0pdd** [comment]
@g4s8 the puzzle [#53](https://github.com/artipie/helm-adapter/issues/53) is still not solved; solved: [#44](https://github.com/artipie/helm-adapter/issues/44), [#52](https://github.com/artipie/helm-adapter/issues/52).

**Comment by 0pdd** [comment]
@g4s8 all 3 puzzles are solved here: [#44](https://github.com/artipie/helm-adapter/issues/44), [#52](https://github.com/artipie/helm-adapter/issues/52), [#53](https://github.com/artipie/helm-adapter/issues/53).
